test/array.jl

```diff
@@ -399,6 +399,19 @@ end
       @test Array(gpu) == reverse(cpu; dims=dim)
     end
 
+    # supports multidimensional reverse
+    if VERSION > v"1.6" 
+      for shape in ([1, 2, 4, 3], [2^5, 2^5, 2^5]),
+          dim in ((1,2),(2,3),(1,3),:)
+        @test testf(x->reverse(x; dims=dim), rand(shape...))
+
+        cpu = rand(shape...)
+        gpu = CuArray(cpu)
+        reverse!(gpu; dims=dim)
+        @test Array(gpu) == reverse(cpu; dims=dim)
+      end
+    end
```

Comment:
CUDA.jl only supports Julia 1.6+

```suggestion
    for shape in ([1, 2, 4, 3], [2^5, 2^5, 2^5]),
        dim in ((1,2),(2,3),(1,3),:)
      @test testf(x->reverse(x; dims=dim), rand(shape...))

      cpu = rand(shape...)
      gpu = CuArray(cpu)
      reverse!(gpu; dims=dim)
      @test Array(gpu) == reverse(cpu; dims=dim)
    end
```